### PR TITLE
Deprecation updates for `BitOps.popcount` and `bigint.mod`

### PR DESCRIPTION
**Pull request by jeremiah-corrado**
Updates for the following deprecations:

- `bigint.mod`: replaced by the standalone `BigInteger.mod` function
- `popcount`: renamed to `popCount`

Adds a compatibility module for bitops. A compatibility module for BigInteger was already set up.

### CODE REVIEW

File: src/compat/gt-130/ArkoudaBitOpsCompat.chpl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module ArkoudaBitOpsCompat {
+  use BitOps;
+
+  proc popCount(x: integral) {
+    return popcount(x);
+  }
+}
```
Comment (bmcdonald3) [inline src/compat/gt-130/ArkoudaBitOpsCompat.chpl:7]: For greater than 1.30, we have the `popCount` method in Chapel I think, so we don't need this overload (i.e., we need the `ArkoudaBitOpsCompat` module for all 3 versions of Chapel, but the one that supports the deprecated feature, we can just have it be an empty module, like https://github.com/Bears-R-Us/arkouda/blob/master/src/compat/gt-130/ArkoudaBigIntCompat.chpl).

Does that make sense? Also possible that I am missing something.

**Review by bmcdonald3 — CHANGES_REQUESTED** [review]

File: src/compat/gt-130/ArkoudaBitOpsCompat.chpl (hunk 1 above)
Comment (jeremiah-corrado) [reply src/compat/gt-130/ArkoudaBitOpsCompat.chpl:7]: Yeah that makes sense, I'll remove that function.

**Review by bmcdonald3 — APPROVED** [review]
Looks good, feel free to promote to a regular PR, thanks Jeremiah!

**Comment by jeremiah-corrado** [comment]
Great, thanks!

**Review by pierce314159 — APPROVED** [review]

**Pull request merged**